lib/src/inline_syntaxes/link_syntax.dart

```diff
@@ -147,18 +160,26 @@ class LinkSyntax extends DelimiterSyntax {
   }
 
   /// Tries to create a reference link node.
+  /// [secondary] indicates whether [label] is inside secondary `[]`
```

Comment:
I don't understand the `secondary` bit here. And it is unused, so can we instead add it in the follow-up Footnote PR?